blocks/career-apply/career-apply.css

```diff
@@ -0,0 +1,73 @@
+.career-apply-wrapper {
```

Comment:
I think, we should be consistent. Either use **rem** or use **px**. If you think you can map the px to nearest whole number rem or to nearest 0.5/0.25 rem then it would be good. For eg. for 19px may be u can use 1.25 rem.

Reply:
As discussed on slack, we agreed to prefer `rem` to `px` but if `px` becomes awkward, we can use `px`.